src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="isAlterationView" id="business-type">
```

Comment:
Does this component apply to both corrections and alterations?

Or is the assumption that this component is only used for alterations? If so then it's a bit confusing due to the component name... which I would then suggest renaming. Thoughts?

Reply:
You are right, we don't correct this (currently). I guess i just followed the pattern of all the other components.
We can change this, although it will result in a single component titled as `AlterSomethingSomething`. 
I'm all for it, if that is what you prefer.

Reply:
Question, is this worth doing at the moment?

Reply:
I think we need to do something to avoid the confusion.

I'm not in favour of having a special folder for alteration components (one potental solution).

This never bothered me before with the v-if in place.

As for renaming, options include:
- ChangeBusinessType
- AlterBusinessType
- BusinessType (but then it should have the v-if to explicitly limit its use)

Up to you.

Reply:
> Question, is this worth doing at the moment?

As reviewer, I ask myself whether a change makes the code better. In this case, I think it raises some confusion.

For expediency, I suggest just restoring the v-if.

Reply:
Ok, i like option 1!

Reply:
The AlterationView getter was also just moved in this instance, to a level up just an fyi.
 More or less the same functionality but didn't want to leave white space in the event it wasn't rendered which was happening in Corrections where it was displaying the section but not the component etc.

Reply:
Updated 👍